src/main/java/org/jabref/logic/protectedterms/ProtectedTermsParser.java

```diff
@@ -29,30 +30,37 @@ public class ProtectedTermsParser {
     private String location;
 
     public void readTermsFromResource(String resourceFileName, String descriptionString) {
-        try {
-            Path path = Path.of(Objects.requireNonNull(ProtectedTermsLoader.class.getResource(Objects.requireNonNull(resourceFileName))).toURI());
-            readTermsList(path);
-            description = descriptionString;
-            location = resourceFileName;
-        } catch (URISyntaxException e1) {
-            LOGGER.error("");
+        description = descriptionString;
+        location = resourceFileName;
+        try (InputStream inputStream = ProtectedTermsLoader.class.getResourceAsStream(Objects.requireNonNull(resourceFileName))) {
```

Comment:
```suggestion
        try (InputStream inputStream = ProtectedTermsLoader.class.getClassLoader().getResourceAsStream(Objects.requireNonNull(resourceFileName))) {
```

Reply:
Works without here.

Reply:
I tried with `./gradlew run` and then protected terms. Worked.